plugins/org.jboss.tools.locus.jsonschema2pojo/0.4.13/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,9 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: org.jboss.tools.locus.jsonschema2pojo.jsonschema2pojo-core
+Bundle-Version: 0.4.13
+Bundle-ClassPath: .
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Bundle-RequiredExecutionEnvironment: JavaSE-1.6

```

Comment:
You'll need some Export-Package directive to make it usable to clients. Also, doesn't this bundle have dependencies?